svgnative/CMakeLists.txt

```diff
@@ -510,6 +515,21 @@ CONFIGURE_FILE("${CMAKE_CURRENT_SOURCE_DIR}/SVGNativeViewerLib.pc.in"
 CONFIGURE_FILE("${CMAKE_CURRENT_SOURCE_DIR}/SVGNativeViewerLib-uninstalled.pc.in"
                "${CMAKE_CURRENT_BINARY_DIR}/SVGNativeViewerLib-uninstalled.pc" @ONLY)
 
+################################
+# setup Google Test for testing
+################################
+if (TESTING)
```

Comment:
Doesn't it practically disable GTest? If we are not going to build it, we should not integrate it and rather concentrate on the essence of this PR.

Which reminds me to ask to update the description of this PR. It is unclear what this PR tries to fix.

Reply:
disable the GTest

Reply:
@tjindal Disabled or enabled? If the tests are to be kept, they should be enabled, right?